src/app/components/ms-identifier/ms-identifier.component.html

```diff
@@ -0,0 +1,74 @@
+<div class="msIdentifier">
+    <evt-ms-desc-section [label]="'msIdentifier'">
+        <div content>   
+            <evt-content-viewer></evt-content-viewer>
+        </div>
+    </evt-ms-desc-section>
+    <div>
+        <evt-ms-desc-section [label]="'place'" [additionalClass]="'settlement'" [inlineLabel]="'inlineLabel'">
```

Comment:
`inlineLabel` is a boolean input. You should pass `true` or `false` to it, not a string! (Check every occurence of it)